### PR TITLE
Update the components colors based on the latest Valet Design System

### PR DESCRIPTION
**Pull request by djalmaaraujo**
## Description

Ps: *Dark mode is not part of the scope*

image

This PR uses the latest develop/tokens/* files and adapts the components using the new color system. All possible components are being updated.

Preview URL: https://deploy-preview-138--vip-design-system-components.netlify.app/

All changes were made in sync with @donaghokeeffe (VIP Designer), matching the latest Figma files from the Valet Design System.

# Steps to test

1. Go to https://deploy-preview-138--vip-design-system-components.netlify.app/
2. Navigate into all non-deprecated components pages
3. Colors have changed. You should not find any accessibility contrast issues 

image

# Steps to test (Optional)

1. Go to the vip-dashboard repo
2. Change the `"@automattic/vip-design-system": "0.27.10",` to `"@automattic/vip-design-system": "github:automattic/vip-design-system#f870605e8312c3b90bd4d5981fc8a1f0c85cc98d"`
3. Do `npm install`
4. Do `npm run dev`
5. Test the dashboard works as expected with the new colors

### CODE REVIEW

File: src/system/Badge/Badge.stories.jsx [hunk 1]
```diff
@@ -8,4 +8,36 @@ export default {
 	component: Badge,
 };
 
-export const Default = () => <Badge>Badge</Badge>;
+const BBadge = props => <Badge { ...props } sx={ { m: 2 } } />;
```
Comment (luiztiago) [inline src/system/Badge/Badge.stories.jsx]: Maybe we can use `const Template` here, such as is used on other files.

File: src/system/Form/Radio.js [hunk 2]
```diff
@@ -10,6 +10,8 @@ import classNames from 'classnames';
 import { Label } from './Label';
 import { screenReaderTextClass } from '../ScreenReaderText/ScreenReaderText';
 import mainTheme from '../theme';
+import { baseControlBorderStyle, inputBaseText } from './Input.styles';
+import { getColor } from '../theme/getColor';
```
Comment (luiztiago) [inline src/system/Form/Radio.js]: Can we remote this line?

**Review by luiztiago — COMMENTED** [review]
Great job. The storybook and UI are much better now.
I suggested only two minor updates.

**Review by pandah3 — COMMENTED** [review]
What's the best way to test this? I'm not sure where to even start 😅

Should we go through all the components in the [Netlify previews](https://deploy-preview-138--vip-design-system-components.netlify.app/) and compare the colors with the expected colors from the [Valet tokens](https://github.com/Automattic/vip-design-system/blob/trunk/tokens/valet-core.json)? This obviously isn't practical so I'm missing something here lol..

**Comment by djalmaaraujo** [comment]
> What's the best way to test this? I'm not sure where to even start 😅
> 
> Should we go through all the components in the [Netlify previews](https://deploy-preview-138--vip-design-system-components.netlify.app/) and compare the colors with the expected colors from the [Valet tokens](https://github.com/Automattic/vip-design-system/blob/trunk/tokens/valet-core.json)? This obviously isn't practical so I'm missing something here lol..

Thanks for the question. You can review all the components in the Netlify preview and make sure they work. We only changed colors; there's no new functionality. 

Some things were removed or deprecated too, but that's minimal, no effect.

I will add a proper way to test in the instructions, but in general, you can also grab the branch into the VIP Dashboard and to some navigation there.

**Review by pandah3 — CHANGES_REQUESTED** [review]
Thank you for the updated testing instructions!

Testing with Netlify, a good chunk of the components have accessibility violations in Dark Mode 😢 . These are the ones I found with the violations:

- Accordion
  - Default
- Button
  - Default
  - Link
  - With on click
- Dropdown
  - Default
  - Complex options
  - (With Dialog is good, no violations)
- Option row
  - Default
- Table
  - Default
- Tabs
  - Default
  - (Set active tab is good, no violations)
- Text
  - Default
- Wizard
  - Default
  - (Custom title is good, no violations)
- Autocomplete
  - Default
  - Inline
  - (Others are good)
- Checkbox
  - Default
- Input
  - Default
- Label
  - Required
  - (Default is good, no violations)
- Radio
  - Default
- Select
  - Default
  - With Group
  - Is Inline
  - With Option Label and Value
  - With On Change
- Textarea
  - Default

**Comment by djalmaaraujo** [comment]
> Thank you for the updated testing instructions!
> 
> Testing with Netlify, a good chunk of the components have accessibility violations in Dark Mode 😢 . These are the ones I found with the violations:
> 
> * Accordion
>   
>   * Default
> * Button
>   
>   * Default
>   * Link
>   * With on click
> * Dropdown
>   
>   * Default
>   * Complex options
>   * (With Dialog is good, no violations)
> * Option row
>   
>   * Default
> * Table
>   
>   * Default
> * Tabs
>   
>   * Default
>   * (Set active tab is good, no violations)
> * Text
>   
>   * Default
> * Wizard
>   
>   * Default
>   * (Custom title is good, no violations)
> * Autocomplete
>   
>   * Default
>   * Inline
>   * (Others are good)
> * Checkbox
>   
>   * Default
> * Input
>   
>   * Default
> * Label
>   
>   * Required
>   * (Default is good, no violations)
> * Radio
>   
>   * Default
> * Select
>   
>   * Default
>   * With Group
>   * Is Inline
>   * With Option Label and Value
>   * With On Change
> * Textarea
>   
>   * Default

Thanks for the reply, but dark mode is not included in the scope of these changes.

**Comment by pandah3** [comment]
Noting that this will break Dark Mode even more, so we will be disabling Dark Mode in a separate PR.
https://a8c.slack.com/archives/CKUJZT08Y/p1671744028394699

**Review by pandah3 — CHANGES_REQUESTED** [review]
Thank you for working on this enormous effort 🎉 
I tested this PR with the [draft Dashboard PR](https://github.com/Automattic/vip-dashboard/pull/2530) using the axe DevTools extension.

Went through all the panels and flows I could in the VIP Dashboard and found some violations of `Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds`:

1. [Database Backups](http://localhost:3000/apps/4018/production/data/database/backups) - [Axe issue](https://axe.deque.com/issues/fe070724-61fe-4658-9108-6b0aba7e791b
)
```
Element has insufficient color contrast of 3.89 (foreground color: #e74135, background color: #fbfbfb, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
```

Screen Shot 2022-12-22 at 5 48 25 PM

2. Media Backups - [Axe issue](https://axe.deque.com/issues/a6d9b72e-c6c4-4d68-94e7-7590e83c6ead)
```
Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
```
Screen Shot 2022-12-22 at 5 51 56 PM

3. Deployments - [Axe issue](https://axe.deque.com/issues/fdd904c2-1937-4ab0-820b-5b490904093e)
Screen Shot 2022-12-22 at 5 54 00 PM

4. Certs - [Axe issue](https://axe.deque.com/issues/71f2c88a-4132-45e6-a73a-2b528785ede9)
```
Element has insufficient color contrast of 2.58 (foreground color: #fbfbfb, background color: #d29137, font size: 10.0pt (13.3333px), font weight: normal). Expected contrast ratio of 4.5:1
```
Screen Shot 2022-12-22 at 5 55 43 PM

5. [Audit log ](http://localhost:3000/apps/5264/production/audit-log?from=2022-12-20T15%3A10%3A37.000Z&eventId=g%3A6617872)- [Axe issue](https://axe.deque.com/issues/e7b864ab-3f43-4222-bf8c-6458308c759d)

Screen Shot 2022-12-22 at 5 58 57 PM

7. Usage - Axe issues [1](https://axe.deque.com/issues/537378c3-1dc0-4077-8c49-2cd9081a1ab9), [2](https://axe.deque.com/issues/d1d3057a-568c-471d-8ef3-46300d8f9789), [3](https://axe.deque.com/issues/03c6bb7b-7384-4085-97da-dc3b9e91949f)

Screen Shot 2022-12-22 at 6 06 23 PM copy

**Comment by djalmaaraujo** [comment]
> Thank you for working on this enormous effort 🎉 I tested this PR with the [draft Dashboard PR](https://github.com/Automattic/vip-dashboard/pull/2530) using the axe DevTools extension.

Thank you for your review @pandah3 

I fixed some issues you mentioned, and some should be fixed in the VIP Dashboard draft PR:

- React Select colors (just a few, because this component is already deprecated), this a PR to replace the one you showed above for the "Type to search…"
- The INFO color (audit log example), is broken, indeed. The color contrast is not the best for the job. I reported to @donaghokeeffe 
- Deployments: Can be fixed in the VIP Dashboard PR (Can be ignored for this PR for now)
- Usage: Can be fixed in the VIP Dashboard PR (Can be ignored for this PR for now)
- Database Backups: Can be fixed in the VIP Dashboard PR (Can be ignored for this PR for now)
- Install Custom Certificate: I found an issue. Fixed here.

**Review by andrea-sdl — APPROVED** [review]
✅ ✅ ✅ ✅ ✅

**Comment by djalmaaraujo** [comment]
Merging, and we will fix the blue color of the Notice once @donaghokeeffe send us an updated version of the Valet Theme

**Pull request merged**